index.html

```diff
@@ -139,55 +139,75 @@ <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
-                <div>
-                    <h2 class="skill-title">Frameworks</h2>
-                    <ul>
-                        <li class="skill-tag">React.js</li>
-                        <li class="skill-tag">Ruby on Rails</li>
-                        <li class="skill-tag">RSpec</li>
-                        <li class="skill-tag">Capybara</li>
-                        <li class="skill-tag">Selenium</li>
-                    </ul>
+                <div class="skills">
+                    <img src="assets/Frameworks.svg" alt="Frameworks">
+                    <div>
+                        <h2 class="skill-title color1">Frameworks</h2>
+                        <ul>
+                            <li class="skill-tag">React.js</li>
+                            <li class="skill-tag">Ruby on Rails</li>
+                            <li class="skill-tag">RSpec</li>
+                            <li class="skill-tag">Capybara</li>
+                            <li class="skill-tag">Selenium</li>
+                        </ul>
+                    </div>
                 </div>
-            </div>
-            <div class="skills">
-                <img src="assets/Skills.svg" alt="Languages">
-                <div>
-                    <h2 class="skill-title">Skills</h2>
-                    <ul>
-                        <li class="skill-tag">Database Management</li>
-                        <li class="skill-tag">Version Control</li>
-                        <li class="skill-tag">CLI</li>
-                        <li class="skill-tag">Web Development</li>
-                        <li class="skill-tag">API Design</li>
-                    </ul>
+                <div class="skills">
+                    <img src="assets/Skills.svg" alt="Languages">
+                    <div>
+                        <h2 class="skill-title color1">Skills</h2>
+                        <ul>
+                            <li class="skill-tag">Database Management</li>
+                            <li class="skill-tag">Version Control</li>
+                            <li class="skill-tag">CLI</li>
+                            <li class="skill-tag">Web Development</li>
+                            <li class="skill-tag">API Design</li>
+                        </ul>
+                    </div>
                 </div>
             </div>
-        </div>
+        </section>
+        <section class="contact">
+                <h2 class="primary color1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="" class="form">
+                <input type="text" placeholder="Full name">
+                <input type="text" placeholder="Email Address">
+                <textarea type="text" class="textarea" placeholder="Hey I'd like to connect too"></textarea>
+                <button type="button" class="get-in-touch-btn" name="get-in-touch" id="">Get in touch</button>
```

Comment:
- [ ] Well done but we have a few points to add before all, you are missing the `formspree` config in your form action
- [ ] All your input files and text area should have a `required` attribute
- [ ] The full name should have a `maxlength="30"` attribute
- [ ] Your text area should have the `maxlength="500"` attribute.
- Please make an effort to adjust your left padding on the placeholder as shown below
<img width="309" alt="1" src="https://github.com/BHObserver/portfolio/assets/77843050/6c4ff8b8-9e04-4407-bc98-163e66d92c38">

FIGMA
<img width="320" alt="2" src="https://github.com/BHObserver/portfolio/assets/77843050/6566b3a9-6fc3-4d1f-a116-590046917dfb">

- [ ] Your button should have `submit` type, not button